trusted-committer/03-keeping-the-community-healthy.asciidoc

```diff
@@ -1,80 +1,76 @@
-community will also be growing.
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] tend to stick around, can spend most of their time developing software, and are able to level up their abilities.
+As a result, a healthy community will be continually growing.
 
 Why do https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] join and stick around in a community? Some do because they
 subscribe to the purpose or the mission of the community. It is the Trusted Committer's job to
 clearly articulate and promote this purpose. The importance of this is often
-not recognized - marketing a community and their products is truly essential.
+not recognized, but marketing a community and its products is truly essential.
 
-Another, more obvious reason for people to stick around is that they enjoy
-working with other members of the community, including the Trusted Committers. In my
-experience, what this comes down to is that community members treat and
-communicate with each other with utmost respect. Contributions are treated as
-like a gift or a donation, rather than something that detracts from their own
-work and they laud excellent and especially first contributions. The Trusted Committer's job
-in all this is primarily to set an example for others, similarly to setting an
-example for the level of software quality that is expected.  If necessary, the
-Trusted Committers are the ones who should create and enact a code of conduct for the
-community. Should there be community members whose behavior is detrimental or
-even toxic to the community health, it is the Trusted Committer's responsibility to either
-try and change or contain this or, in the worst case, to remove people from the
-community. Trusted Committers should create opportunities for people to get together
-regularly and get to know each other personally.
+Another, more obvious reason for people to stick around is that they
+enjoy working with other members of the community, including the Trusted Committers. A thriving community is one where members treat
+and communicate with each other with the utmost respect. Contributions are
+treated like gifts or donations, and excellent (especially
+first) contributions are lauded. The Trusted Committer’s job in all this is primarily to set an
+example for others, similar to setting an example for the level of
+software quality expected. If necessary, the Trusted Committers are the ones
+who should create and enact a code of conduct for the community. If
+there are community members whose behavior is detrimental or toxic to
+the community’s health, it is the Trusted Committer’s responsibility to address this. 
+Trusted Committers should create opportunities for people to get together
+regularly and get to know each other personally and to peacefully resolve conflicts as they arise.
```

Comment:
I appreciate the "resolve conflicts" part here, however one could read it as if resolution of conflicts is only possible in person or should be done in person. 
It can in fact often help to do so, however is in my view not mandatory. Or even possible in many remote / distrib conditions.

Reply:
I see this as a continuation of the "address conflicts" line with an attempt to show an example of addressing conflicts.
To your immediate question, "get together regularly and get to know each other personally" does not mean actually meeting "in person", aka being physically in the same location. Our weekly Learning Path calls are a great example of what I believe is expressed in this line.
I agree that meetings are not strictly necessary when working on the code, but I will be the first one to call a meeting as soon as I hear anyone complaining on another person's behavior. And I will start that meeting with a great deal of ice breaker, have people talk about themselves, then about their daily work, and only THEN about the conflict.

TL;DR: 
(1) I like the line as it is. 
(2) Specifically, "getting together" does not mean "in person". 
(3) If I convinced you, please confirm, otherwise please let me know what you'd like to change.

Reply:
I am entirely on board with you here. Especially also with the conflict resolution approach. 
Since we aim this at people who might not be excessively used to all this remote / "not necessarily f2f" stuff I think we should add a little hint that the "personally" part can also be remote.
I have added a commit suggestion here. 

```suggestion
regularly and get to know each other personally and to peacefully resolve conflicts as they arise.
This can, but does not have to be, in the same physical location. Even regular video or even audio-only calls can have the desired positive effects.
```